lms/djangoapps/learner_recommendations/views.py

```diff
@@ -179,7 +181,7 @@ def get(self, request, course_id):
 
         for course in unrestricted_courses:
             course.update({
-                "active_course_run": course.get("course_runs")[0]
+                "active_course_run": get_active_course_run(course)
             })
```

Comment:
Do you think we should only return courses that have active course runs? 

Draft code of what it could look like
```python
unrestricted_courses = []

for course in filtered_courses:
    if _has_country_restrictions(course, user_country_code):
        continue

    active_course_run = get_active_course_run(course)
    if active_course_run:
        course.update({"active_course_run": active_course_run})
        unrestricted_courses.append(course)
```

Reply:
Yes, this does look like something we should be doing. I'll definitely look into this 👍